### DISCUSSION TITLE
How to create a subdomain in AWS that points to an open room?

### DISCUSSION THREAD

**Discussion by ro6ertkenny** [mozilla/hubs#3113] [Hubs Cloud Questions] [answered]
Has anyone tried to create a subdomain in an AWS Route 53 hosted zone that points to an open room?  I've exhaused all the aws documentation and tried creating every AName / CName pointing every which way ... with no avail ... and the CloudFormation stack using 2 domains for the internal and external makes it even more perplexing

**Comment by robinkwilson** [comment, marked as answer]
I haven't heard of anyone making a subdomain pointing to an open room. I do know that there's a few code snippets to siphon the person to an open room (community made): https://gist.github.com/Utopiah/8bf9375ef73d46b6c471c3ec2abae405 or https://gist.github.com/camelgod/a712ce7d43f21260ae5b4971b9a58dfb

If you can get the subdomain to direct the person to your homepage with query params, you could deploy a [custom client](https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html) that will see that query params and forward that user to an open room. As a potential design. 

NOTE: we're actively updating the room API's (https://github.com/mozilla/hubs/issues/3062) so those snippets could be out of date in the near future.
